eng/tools/BaselineGenerator/Program.cs

```diff
@@ -30,16 +31,18 @@ static void Main(string[] args)
             new Program().Execute(args);
         }
 
-        private readonly CommandOption _source;
+        private readonly CommandOption _sources;
         private readonly CommandOption _output;
         private readonly CommandOption _update;
 
+        private static readonly string[] _defaultSources = new string[] { "https://api.nuget.org/v3/index.json" };
+
         public Program()
         {
-            _source = Option(
-                "-s|--package-source <SOURCE>",
-                "The NuGet source of packages to fetch",
-                CommandOptionType.SingleValue);
+            _sources = Option(
+                "-s|--package-sources <SOURCE>...",
```

Comment:
My bad, this version of `CommandLineUtils` doesn't support the `...` indicator that more than one is allowed.
```suggestion
                "-s|--package-sources <Sources>",
```